src/model/ThermalZone.hpp

```diff
@@ -541,6 +541,17 @@ namespace model {
 
     std::vector<AirLoopHVAC> airLoopHVACs() const;
 
+    // SQL Queries
+    boost::optional<double> getAutosizedValueFromZoneSizes(const std::string& columnName, const std::string& loadType) const;
+
+    boost::optional<double> autosizedMaximumOutdoorAirFlowRate() const;
+    boost::optional<double> autosizedMinimumOutdoorAirFlowRate() const;
+    boost::optional<double> autosizedCoolingDesignAirFlowRate() const;
+    boost::optional<double> autosizedHeatingDesignAirFlowRate() const;
+    boost::optional<double> coolingDesignLoad() const;
+    boost::optional<double> designAirFlowRate() const;
+    boost::optional<double> heatingDesignLoad() const;
```

Comment:
I'm not sold on the naming. `autosizedCoolingDesignLoad` would be a bit weird too probably.
Has someone got an idea to clearly convey the idea that this is a grab from the sql?

@asparke2 @tijcolem @kbenne @joseph-robertson

Reply:
I think that autosizedCoolingDesignLoad, etc. is good. Any thing that is "autosizedFoo" comes from the SQL file

Reply:
Renamed in 4f7cd195ee54b56ab0c4070d670f0d307c46435d